cli/cmd/flags.go

```diff
@@ -64,19 +66,37 @@ func bindRPCURL(cmd *cobra.Command, rpcURL *string) {
 func bindUnjailConfig(cmd *cobra.Command, cfg *unjailConfig) {
 	netconf.BindFlag(cmd.Flags(), &cfg.Network)
 	bindPrivateKeyFile(cmd, &cfg.PrivateKeyFile)
-	_ = cmd.MarkFlagRequired("network")
+	_ = cmd.MarkFlagRequired(flagNetwork)
+}
+
+func bindEOAConfig(cmd *cobra.Command, cfg *eoaConfig) {
+	bindPrivateKeyFile(cmd, &cfg.privateKeyFile)
+	netconf.BindFlag(cmd.Flags(), &cfg.network)
+
+	cmd.Flags().StringVar(&cfg.evmRPC, "evm-rpc", "", "Optional Omni EVM execution RPC API endpoint. Defaults to consensus.<network>.omni.network")
+
+	_ = cmd.MarkFlagRequired(flagNetwork)
+}
+
+func bindDelegateConfig(cmd *cobra.Command, cfg *delegateConfig) {
+	bindEOAConfig(cmd, &cfg.eoaConfig)
+	const flagSelf = "self"
+	cmd.Flags().Uint64Var(&cfg.amount, flagDelegationAmount, cfg.amount, "Increase delegation amount in OMNI (minimum 1 OMNI)")
+	cmd.Flags().BoolVar(&cfg.self, flagSelf, true, "Mark self-delegation")
```

Comment:
lets do default false, since this will be the default in the future, so then we don't need to change defaults later.

Reply:
also maybe change the comment: `Enables self-delegation setting target validator address to provided private key`